src/components/Accordion/AccordionTitle.tsx

```diff
@@ -66,7 +66,7 @@ class AccordionTitle extends UIComponent<ReactProps<AccordionTitleProps>, any> {
 
     return (
       <ElementType {...rest} className={classes.root} onClick={this.handleClick}>
-        {active ? <span>&#9660;</span> : <span>&#9654;</span>}
+        {active ? <span>&#x25BE;</span> : <span>&#x25B8;</span>}
```

Comment:
Can we move the Unicode characters under some variables with descriptive naming?
for e.g. `const arrowDown = '&#x25BE;'`
If firstly look at code without running, can be a "guess" game to know what does it represent :)

Reply:
I moved all usages of the unicode characters in the styles and added constants in the teams theme.